website/docs/advanced/proof-of-stake-devnet.md

```diff
@@ -105,12 +105,12 @@ You should see an output showing you the version you have installed. Next, creat
     mkdir devnet && cd devnet
     
 
-The instructions below are running with Prysm commit [cb9b5e8f6e91adc8c6cdb2ca39708703e88c0b63](https://github.com/prysmaticlabs/prysm/commit/cb9b5e8f6e91adc8c6cdb2ca39708703e88c0b63) and go-ethereum commit [890e2efca2111c790c6d5eb55e29981816ee1fe9](https://github.com/ethereum/go-ethereum/commit/890e2efca2111c790c6d5eb55e29981816ee1fe9)
+The instructions below are running with Prysm commit [be4ef54482025b216554e6a5ff0c6e3d1a398d0b](https://github.com/prysmaticlabs/prysm/commit/be4ef54482025b216554e6a5ff0c6e3d1a398d0b) and go-ethereum commit [c8a6b7100c56255a9cde580be59136beb8d28b8e](https://github.com/ethereum/go-ethereum/commit/c8a6b7100c56255a9cde580be59136beb8d28b8e)
```

Comment:
do we need to run with a specific commit? why can't this be develop ( if things are merged in)

Reply:
The main reason was so that we could say it definitely works at this commit. But the docker version is using latest anyway so I don't see why we can't on the manual setup